sound/soc/sof/topology.c

```diff
@@ -2542,6 +2542,9 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 
 	w->dobj.private = swidget;
 	list_add(&swidget->list, &sdev->widget_list);
+
+	swidget->ref_count++;
```

Comment:
are these changes to `.ref_count` protected everywhere?

Reply:
I guess they are, but same thing popped to my mind. Calling tihs specifically "ref_count" will get people's attention for sure.

Reply:
i've changed it to atomic_t now. Would that be enough @lyakh?